source/NVDAObjects/window/edit.py

```diff
@@ -773,13 +730,10 @@ def updateSelection(self):
 class Edit(EditableTextWithAutoSelectDetection, Window):
 
 	editAPIVersion=0
-	editAPIUnicode=True
-	# #4291: Use ITextDocument in Windows 7 and later, as it's very slow in earlier versions.
-	useITextDocumentSupport=(winVersion.winVersion.major,winVersion.winVersion.minor)>=(6,1)

```

Comment:
@MichaelDCurran: the useITextDocumentSupport property seems to have been there to limit the support of ITextDocument to windows 7 and abbove. Since we aren't longer supporting Windows versions older than 7, this code now became obsolete (i.e. this variable would be always True). That's also the reason why I removed it from the IAccessible windowless rich text edit support as you saw.